newshandler/src/main/java/life/qbic/newshandler/usermanagement/email/EmailSubmissionService.java

```diff
@@ -78,6 +79,16 @@ public void send(Email email) {
     }
   }
 
+  @Override
+  public void send(String recipientAddress, String recipientFullName, String subject, String message) {
+    var email = new Email(message, subject, "no-reply@qbic.life",
+        new Recipient(recipientAddress, recipientFullName),  "text/plain");
+    sendPlainEmail(email).ifSuccessOrElse(
+            successResponse -> reportSuccess(successResponse, email),
+            failureResponse -> reportFailure(failureResponse, email)
+    );
+  }
```

Comment:
Unnecessary if we stick to the Email DTO which enables us to avoid redundancy here :) 
```suggestion
```